docs/README.md

```diff
@@ -1,9 +1,15 @@
 # WireViz
 
+
+[![PyPI - Version](https://img.shields.io/pypi/v/wireviz.svg?colorB=blue)](https://pypi.org/project/wireviz/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/wireviz.svg?)](https://pypi.org/project/wireviz/)
+[![PyPI - Downloads](https://img.shields.io/pypi/dm/wireviz)](https://pypi.org/project/wireviz/)
```

Comment:
Do you want any other badges from https://shields.io/
- https://img.shields.io/github/issues/formatc1702/WireViz
- https://img.shields.io/github/forks/formatc1702/WireViz
- https://img.shields.io/github/stars/formatc1702/WireViz
- https://img.shields.io/github/license/formatc1702/WireViz
- https://img.shields.io/twitter/url?url=https%3A%2F%2Fgithub.com%2Fformatc1702%2FWireViz%2F

Reply:
Thanks. Since issues, forks, stars and license are already clearly displayed on the project's homepage, I don't think this adds too much valuable information.